<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountWindow</class>
 <widget class="QMainWindow" name="AccountWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <widget class="QLineEdit" name="UserNameLe"/>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QPushButton" name="AdminBtn">
      <property name="text">
       <string>مدیر</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QPushButton" name="LoginBtn">
      <property name="text">
       <string>ورود کاربر</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="PassLbl">
      <property name="text">
       <string>رمز عبور</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="PassLe">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="UserNameLbl">
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>نام کاربری</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QPushButton" name="SignUpBtn">
      <property name="text">
       <string>ثبت نام کاربر</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="UserSignInBtn">
      <property name="text">
       <string>ورود</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>تنظیمت</string>
    </property>
    <addaction name="ReturnBtn"/>
    <addaction name="separator"/>
    <addaction name="ExitBtn"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ReturnBtn">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/Icons/return_icon_154912.png</normaloff>:/Icons/Icons/return_icon_154912.png</iconset>
   </property>
   <property name="text">
    <string>بازگشت</string>
   </property>
  </action>
  <action name="ExitBtn">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/Icons/exit-icon-png-close-0.png</normaloff>:/Icons/Icons/exit-icon-png-close-0.png</iconset>
   </property>
   <property name="text">
    <string>خروج</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ExitBtn</sender>
   <signal>triggered()</signal>
   <receiver>AccountWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ReturnBtn</sender>
   <signal>triggered()</signal>
   <receiver>AccountWindow</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
